README.md

```diff
@@ -98,42 +98,44 @@ https://arewecompressedyet.com/analyzer/?d=https://people.xiph.org/~mbebenita/an
 
 # Contributing
 
-## Coding style
-Check code formatting with [rustfmt](https://github.com/rust-lang-nursery/rustfmt) before submitting a PR.
-rav1e currently uses the nightly version of rustfmt.
-
-To install nightly:
+## Toolchain
+rav1e uses the stable version of Rust (the stable toolchain)
 
+To install the toolchain:
 ```
-rustup install nightly
+rustup install stable
 ```
 
-To install the nightly version of rustfmt:
+
+## Coding style
+Check code formatting with [rustfmt](https://github.com/rust-lang-nursery/rustfmt) before submitting a PR.
+
+To install the rustfmt:
 
 ```
-rustup component add rustfmt-preview --toolchain nightly
+rustup component add rustfmt
 ```
 
 then
 
 ```
-cargo +nightly fmt -- --check
+cargo fmt -- --check
 ```
 
-You should also try [clippy](https://github.com/rust-lang-nursery/rust-clippy).
-Rust also uses nightly for clippy.
+
+## Code Analysis
+The [clippy](https://github.com/rust-lang-nursery/rust-clippy) will help catch common mistakes and improve your Rust code.
+
+We also recommend you use it before submitting a PR.
 
 To install clippy:
 
 ```
-rustup component add clippy-preview --toolchain nightly
+rustup component add clippy
 ```
 
-then
+then you can search "cargo clippy" in  [travis script](https://github.com/xiph/rav1e/blob/master/.travis.yml) for detailed command and run it in your local machine.
```

Comment:
It would be nice to see the file name of the "travis script", without the need to hover over the link or click it. I'd personally replace "travis script" with just ".travis.yml".

"in your local machine" could be simplified to "on your machine".